invenio.cfg

```diff
@@ -176,4 +176,104 @@ OAUTHCLIENT_AUTO_REDIRECT_TO_EXTERNAL_LOGIN = False  # autoredirect to external
+            "type": "nested",
+            "properties": {
+                "subject": {"type": "keyword"},
+                "object": {"type": "keyword"},
+            },
+        }
+
+    @property
+    def field(self):
+        # For example see "Host of" on https://zenodo.org/record/3949282
+        return fields.List(fields.Nested({
+            'subject': fields.List(SanitizedUnicode(required=True)),
+            'object': fields.List(SanitizedUnicode(required=True)),
+        }))
+
+
+
+RDM_NAMESPACES = {
+    "dwc": "http://rs.tdwg.org/dwc/terms/",
+    "gbif-dwc": "http://rs.gbif.org/terms/1.0/",
+    "ac": "http://rs.tdwg.org/ac/terms/",
+    "openbiodiv": "http://openbiodiv.net/ontology#",
+    "obo": "http://purl.obolibrary.org/obo/",
+    "dc": "http://purl.org/dc/terms/",
+}
+
+
+RDM_CUSTOM_FIELDS = [
+    # dwc
+    KeywordCF(name='dwc:basisOfRecord', multiple=True),
+    KeywordCF(name='dwc:catalogNumber', multiple=True),
+    KeywordCF(name='dwc:class', multiple=True),
+    KeywordCF(name='dwc:collectionCode', multiple=True),
+    KeywordCF(name='dwc:country', multiple=True),
+    KeywordCF(name='dwc:county', multiple=True),
+    KeywordCF(name='dwc:dateIdentified', multiple=True),
+    KeywordCF(name='dwc:decimalLatitude', multiple=True),
+    KeywordCF(name='dwc:decimalLongitude', multiple=True),
+    KeywordCF(name='dwc:eventDate', multiple=True),
+    KeywordCF(name='dwc:family', multiple=True),
+    KeywordCF(name='dwc:genus', multiple=True),
+    KeywordCF(name='dwc:identifiedBy', multiple=True),
+    KeywordCF(name='dwc:individualCount', multiple=True),
+    KeywordCF(name='dwc:institutionCode', multiple=True),
+    KeywordCF(name='dwc:kingdom', multiple=True),
+    KeywordCF(name='dwc:lifeStage', multiple=True),
+    KeywordCF(name='dwc:locality', multiple=True),
+    KeywordCF(name='dwc:materialSampleID', multiple=True),
+    KeywordCF(name='dwc:namePublishedInID', multiple=True),
+    KeywordCF(name='dwc:namePublishedInYear', multiple=True),
+    KeywordCF(name='dwc:order', multiple=True),
+    KeywordCF(name='dwc:otherCatalogNumbers', multiple=True),
+    KeywordCF(name='dwc:phylum', multiple=True),
+    KeywordCF(name='dwc:preparations', multiple=True),
+    KeywordCF(name='dwc:recordedBy', multiple=True),
+    KeywordCF(name='dwc:scientificName', multiple=True),
+    TextCF(name='dwc:scientificNameAuthorship', multiple=True),
+    KeywordCF(name='dwc:scientificNameID', multiple=True),
+    KeywordCF(name='dwc:sex', multiple=True),
+    KeywordCF(name='dwc:specificEpithet', multiple=True),
+    KeywordCF(name='dwc:stateProvince', multiple=True),
+    KeywordCF(name='dwc:taxonID', multiple=True),
+    KeywordCF(name='dwc:taxonRank', multiple=True),
+    KeywordCF(name='dwc:taxonomicStatus', multiple=True),
+    KeywordCF(name='dwc:typeStatus', multiple=True),
+    KeywordCF(name='dwc:verbatimElevation', multiple=True),
+    KeywordCF(name='dwc:verbatimEventDate', multiple=True),
```

Comment:
minor: iso date?

Reply:
Not for this one, because its `verbatim` (so could also be something like "1st of May, 1996"). There's also `dwc:eventDate` which I put as ISO date though.